packages/nextjs/src/FronteggProvider.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { FC, useCallback, useEffect, useMemo } from 'react';
+import { FronteggConfigOptions, initialize } from '@frontegg/admin-portal';
+import { FronteggProvider as ReduxProvider } from '@frontegg/react-hooks';
+import { ContextHolder, RedirectOptions } from '@frontegg/rest-api';
+import { NextRouter, useRouter } from 'next/router';
+
+type ConnectorProps = FronteggConfigOptions & {
+  router: NextRouter;
+};
+
+export const Connector: FC<ConnectorProps> = ({ router, ...props }) => {
+  const isSSR = typeof window === 'undefined';
+
+  // v6 or v5
+  const baseName = props.basename ?? router.basePath;
+
+  const onRedirectTo = useCallback((_path: string, opts?: RedirectOptions) => {
+    let path = _path;
+    if (path.startsWith(baseName)) {
+      path = path.substring(baseName.length);
+    }
+    if (opts?.preserveQueryParams) {
+      path = `${path}${window.location.search}`;
+    }
+    if (opts?.refresh && !isSSR) {
+      // @ts-ignore
+      window.Cypress ? router.push(path) : (window.location.href = path);
+    } else {
+      opts?.replace ? router.replace(path) : router.push(path);
+    }
+  }, []);
+
+  const app = useMemo(() => {
+    if (typeof window === 'undefined') {
```

Comment:
It looks like this is the main change compare to React, right? just interested to make sure I understand

Reply:
No, we have other library for routing